### DISCUSSION TITLE
Support IPv6 only

### DISCUSSION THREAD

**Discussion by benkeil** [mysticaltech/terraform-hcloud-kube-hetzner#1446] [Ideas]
### Description

To also save the costs of ipV4 addresses, it would be nice to have ipV6 only cluster.

**Comment by BrammyS** [comment]
Would be a really nice thing to have! :)

**Comment by janfrederik** [comment]
The discussion in #1255 has similar ideas. Maybe close this one?

**Comment by mysticaltech** [comment]
Now possible, you can disable ipv4.
